--- tlpkg/TeXLive/TLDownload.pm
+++ tlpkg/TeXLive/TLDownload.pm
@@ -48,6 +48,7 @@ sub new
     env_proxy => 1,
     timeout => $TeXLive::TLConfig::NetworkTimeout,
   );
+  $ua->ssl_opts(verify_hostname => 0, SSL_verify_mode => 0x00);
   $self->{'ua'} = $ua;
   $self->{'enabled'} = 1;
   $self->{'errorcount'} = 0;
--- tlpkg/TeXLive/TLConfig.pm
+++ tlpkg/TeXLive/TLConfig.pm
@@ -118,6 +118,7 @@ our %FallbackDownloaderArgs = (
              '--fail', '--location',
              '--connect-timeout', "$NetworkTimeout", '--silent', '--output'],
   'wget' => ['--user-agent=texlive/wget', '--tries=10',
+             '--no-check-certificate',
              "--timeout=$NetworkTimeout", '-q', '-O'],
 );
 # the way we package things on the web
